src/sys.c

```diff
@@ -156,16 +170,28 @@ static const JSCFunctionListEntry tjs_sys_funcs[] = {
     TJS_CGETSET_DEF("exepath", tjs_exepath, NULL),
 };
 
+static const JSCFunctionListEntry tjs_gc_funcs[] = {
+    TJS_CFUNC_DEF("run", 0, js_std_gcRun),
+    TJS_CFUNC_DEF("setThreshold", 1, js_std_gcSetThreshold),
+    TJS_CFUNC_DEF("getThreshold", 0, js_std_gcGetThreshold)
+};
+
 void tjs__mod_sys_init(JSContext *ctx, JSValue ns) {
     JS_SetPropertyFunctionList(ctx, ns, tjs_sys_funcs, countof(tjs_sys_funcs));
     JS_DefinePropertyValueStr(ctx, ns, "args", tjs__get_args(ctx), JS_PROP_C_W_E);
     JS_DefinePropertyValueStr(ctx, ns, "version", JS_NewString(ctx, tjs_version()), JS_PROP_C_W_E);
+    
     JSValue versions = JS_NewObjectProto(ctx, JS_NULL);
     JS_DefinePropertyValueStr(ctx, versions, "quickjs", JS_NewString(ctx, JS_GetVersion()), JS_PROP_C_W_E);
     JS_DefinePropertyValueStr(ctx, versions, "tjs", JS_NewString(ctx, tjs_version()), JS_PROP_C_W_E);
     JS_DefinePropertyValueStr(ctx, versions, "uv", JS_NewString(ctx, uv_version_string()), JS_PROP_C_W_E);
     JS_DefinePropertyValueStr(ctx, versions, "curl", JS_NewString(ctx, curl_version()), JS_PROP_C_W_E);
     JS_DefinePropertyValueStr(ctx, versions, "wasm3", JS_NewString(ctx, M3_VERSION), JS_PROP_C_W_E);
+
+    JSValue gc = JS_NewObjectProto(ctx, JS_NULL);
+    JS_SetPropertyFunctionList(ctx, gc, tjs_gc_funcs, countof(tjs_gc_funcs));
+    JS_DefinePropertyValueStr(ctx, ns, "$gc", gc, 0);
```

Comment:
Please call this `_gc` so it doesn't get auto-exported to the `tjs` global.

Reply:
From my tests because of the 0 in JS_DefinePropertyValueStr it is not exposed.
I can change it but I would like to understand the mechanism.
Is there an automatic operation I missed which is hiding symbols starting with `_`?

Reply:
Here is where `_` prefixed things are hidden: https://github.com/saghul/txiki.js/blob/1d936302faa9c6342c964aa9b0bbeb547faf0b77/src/js/core/index.js#L56

Reply:
Thank you for the handle, I missed it entirely.

Reply:
Done